### PR TITLE
Add: ability to use several setting profiles using startup flag

### PR DESCRIPTION
**Pull request by omkelderman**
as discussed in #117

### CODE REVIEW

**Comment by omkelderman** [comment]
I uh, may have committed this with my work email instead of my normal gmail I use for github lmao, just noticed that xD

**Comment by omkelderman** [comment]
took me two force pushes lol, but should be fixed now

**Comment by Piotrekol** [comment]
Some filename validation would be nice 
```
Could not find a part of the path 'D:\Programy\StreamCompanion\settings.somethin\.ini'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
```
either don't allow SC to start with name that can't be saved, or remove invalid filename characters eg. https://github.com/Piotrekol/StreamCompanion/blob/171fab829d921aa9bae355904d8bb90c0eba0e47/osu!StreamCompanion/Code/Modules/osuFallbackDetector/OsuFallbackDetector.cs#L72-L79

**Comment by omkelderman** [comment]
uh ye, gud call lol, kinda forgot about that oops

will do later, gotta go now :eyes: (or if you feel like it, feel free to do it xd, you should be able to push to this branch)

File: osu!StreamCompanion/Program.cs [hunk 1]
```diff
@@ -28,7 +29,13 @@ static void Main(string[] args)
             string appGuid = ((GuidAttribute)Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(GuidAttribute), false).GetValue(0)).Value.ToString();
             string mutexId = string.Format("Global\\{{{0}}}", appGuid);
 
-            string settingsProfileName = GetSettingsProfileNameFromArgs(args);
+            string settingsProfileName = GetSettingsProfileNameFromArgs(args)?.Trim();
+            if(settingsProfileName.IndexOfAny(Path.GetInvalidFileNameChars()) >=0)
```
Comment (Piotrekol) [inline osu!StreamCompanion/Program.cs]: Doesn't boot when profile name is not provided(launched by double-clicking exe)

Comment (omkelderman) [reply osu!StreamCompanion/Program.cs]: im dumb smh, should be ez fix, will do soon

**Comment by omkelderman** [comment]
that should fix that, want me to rebase this mess into one commit? or are you just gonna squash and merge

**Pull request merged**